src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -41,6 +41,8 @@ public static class AspNetWebApi2Integration
         private static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(nameof(IntegrationIds.AspNetWebApi2));
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(AspNetWebApi2Integration));
 
+        private static bool CanReadResponseHeaders = true;
```

Comment:
I don't see this field being used anywhere.

Reply:
Good catch, this got refactored out. Fixed in f9debd5b8032b91f44e8d852ac9f8f7c475cec3a